drivers/temperature/adt7420/adt7420.h

```diff
@@ -147,6 +203,9 @@ void adt7420_set_operation_mode(struct adt7420_dev *dev,
 void adt7420_set_resolution(struct adt7420_dev *dev,
 			    uint8_t resolution);
 
+/*! Resets the SPI or I2C inteface for the ADT7420/ADT7320*/
+void adt7420_reset_interface(struct adt7420_dev *dev);
+
```

Comment:
Can you do this modification in place? I mean on the adt7420_reset function, not delete that one and create a new one.

Reply:
Anyway changing function name is really needed?

Reply:
yeah, i agree, please use the old name, no reason to change to a lenghtier one

Reply:
Name has been changed back to the original